docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -51,6 +51,15 @@ are also available. They are discussed later in this topic.
   - Enable caching for partition metadata. You can disable caching to avoid
     inconsistent behavior that results from it.
   - `true`
+* - `hive.metastore-cache.cache-missing`
+  - Enable metadata caching for missing tables.
```

Comment:
whats a missing table? I think we need to describe this more.

Reply:
This is about caching absence of objects, i.e., whether we will remember that an object was not found and avoid going back to metstore for it, or always ask metastore for it as long as the object remains not found.